app/views/shopify_app/sessions/request_storage_access.html.erb

```diff
@@ -0,0 +1,395 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <base target="_top">
+  <title>Redirecting…</title>
+  <style>
+    html,
```

Comment:
I'm rethinking my suggestion about inlining the styles since we have to do it three times now.

Reply:
can we put them in a partial? reusability + inlining